build.gradle

```diff
@@ -1,16 +1,18 @@
 buildscript {
-    apply from: "$rootDir/gradle/buildscript.gradle", to: buildscript
+	apply from: "$rootDir/gradle/buildscript.gradle", to: buildscript
 }
 
-group = 'com.github.node-gradle'
+plugins {
+	id('idea')
+	id('groovy')
+	id('maven-publish')
+	id('org.gradle.kotlin.kotlin-dsl') version "1.3.1"
+	id('com.cinnober.gradle.semver-git') version "2.4.0"
```

Comment:
note to self: see if we have to move the configuration for semver-git when we move to `plugins {}`